addons/web_editor/static/src/js/tours/rte.js

```diff
@@ -323,6 +323,6 @@ tour.register('rte_inline', {
     trigger: '#wrapwrap img:eq(1)[height]:not([class*="fa"])',
 }, {
     content: "check the second font class to css",
-    trigger: '#wrapwrap img:eq(2)[style*="float: right"]',
+    trigger: '#wrapwrap img:eq(2)[style*="float: right"],[style*="float:right"]',
```

Comment:
oh, I think that the selector is wrong. It should be 
`#wrapwrap img:eq(2)[style*="float: right"],#wrapwrap img:eq(2)[style*="float:right"]`

The comma separator splits this into 2 different rules

Reply:
Right, It was working just because there is only one element with float:right style.
Thanks for the review Ged.